security-oauth2/src/main/java/io/micronaut/security/oauth2/configuration/OauthClientConfigurationProperties.java

```diff
@@ -25,29 +25,17 @@
 import io.micronaut.http.MediaType;
 import io.micronaut.security.oauth2.client.clientcredentials.ClientCredentialsConfiguration;
 import io.micronaut.security.oauth2.client.clientcredentials.propagation.ClientCredentialsHeaderTokenPropagatorConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.AuthorizationEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.DefaultEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.DefaultSecureEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.EndSessionEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.EndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.IntrospectionEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.RevocationEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.SecureEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.TokenEndpointConfiguration;
+import io.micronaut.security.oauth2.configuration.endpoints.*;
```

Comment:
are the use of these wildcard imports intentional?

Reply:
It was the IDE optimising them. Wildcard imports are allowed as per our Checkstyle config.